dbfit-java/.gitignore

```diff
@@ -1,2 +1,3 @@
 download
 *.log
+*/bin
```

Comment:
What is using this `/bin` folder? Is it auto-created by gradle or something else?

Reply:
Yes, it is autocreated and I wondered why this was not the case before.

Reply:
Java 7 is out of support already. I'm not sure if we can make it compatible to Java 7. At least we should be be compatible to Java 11 which is the current LTS Java version as it seems (better would be Java 8).
[Java Longterm Support Roadmap](https://www.oracle.com/java/technologies/java-se-support-roadmap.html)
[OpenJDK Support](https://adoptopenjdk.net/support.html)

Reply:
I've added Java 8 to the travis yml .

Reply:
Java 8 Build Error: Expected feature release number in range of 9 to 16, but got: 8, so I guess the codebase currently is compatible to JAVA 9.

Reply:
Issue resolved. Travis distro was changed (trusty -> xenial) and introduced this problem. I've added a config to the yml and now JDK 8 compiles well. Will try JDK 7 next.

Reply:
JDK7 doesn't compile even with gradle so I think JDK8 is best we can get

Reply:
Newer versions of gradle (since version 6 I think) are not compatible with JDK 7 as far as I remember.

Current master branch used to build fine with jdk7. So I guess we should be able to make it work under it.

Reply:
I've upgraded to Gradle 6 which needs jdk8 to run. https://docs.gradle.org/current/userguide/compatibility.html
I can try to compile it with Jdk 7 but this requires to have jdk7 installed in parallel. I think jdk 7 is out of support and hardly anyone will use it in production anymore. We should invest our time on something more productive and useful for users.

Reply:
@bhuesemann, fair enough. Let's drop the JDK7 support from dbfit 4.0, it's anyway a major release.

I suggest that we create a separate pull request for the java version updates only - I mean, moving to JDK8 as default target.

Reply:
@bhuesemann, fair enough. Let's make JDK8 as main target and drop jdk7 support. Let's make it in a separate pull request dedicated for Java version update related stuff only.